lib/stages/SampleStage.cpp

```diff
@@ -0,0 +1,67 @@
+#include "SampleStage.hpp"
+
+#include "StageFactory.hpp"
+#include "errors.h"
+#include "visUtil.hpp" // for frameID, modulo
+
+/* Load the classes we will be using
+ * TODO: Annotate the includes and usings */
+
+using kotekan::bufferContainer;
+using kotekan::Config;
+using kotekan::Stage;
+
+// Register the stage with abstract factory.
+// Needed for Kotekan to be able to find the stage.
+REGISTER_KOTEKAN_STAGE(SampleStage);
+
+/* Constructor for the stage
+    TODO: Annotate the arguments everywhere
```

Comment:
I would include these annotations in the header using the doxygen standard to keep things consistent.